caravel/assets/javascripts/explorev2/components/charts/TimeSeriesLineChart.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { PropTypes } from 'react';
+import * as V from 'victory';
+import theme from '../../../components/VictoryTheme';
+import moment from 'moment';
+import { schemeCategory20c } from 'd3-scale';
+import Legend from './Legend';
+
+const propTypes = {
+  data: PropTypes.array.isRequired,
+  label1: PropTypes.string.isRequired,
+  height: PropTypes.number.isRequired,
+};
+
+export default class TimeSeriesLineChart extends React.Component {
+  constructor(props) {
+    super(props);
+    this.keysToColorsMap = this.mapKeysToColors(props.data);
+  }
+
+  mapKeysToColors(data) {
+    // todo: what if there are more lines than colors in schemeCategory20c?
```

Comment:
I believe that d3 performs `i % lengthOfCategoryArray` so that it just re-uses colors. still annoying that colors are re-used, need magic from @elibrumbaugh